.gitattributes

```diff
@@ -0,0 +1,15 @@
+# Path-based git attributes
```

Comment:
So, I feel it's more valuable that exports and archives from GitHub.com are a complete identical copy to what is in the work tree. This file also references paths like psalm.xml and .travis.yml which don't exist. 

Given that this breaks the integrity of archives, I would not accept this contribution.